src/views/Login.vue

```diff
@@ -67,16 +67,16 @@ export default {
       this.error = null
 
       try {
-        const result = await axios.get('/api/ping', {
-          headers: { Authorization: `Bearer ${this.token}` }
-        })
+        const result = await axios.post('/admin/login?password=' + this.token)
 
-        if (result.data === 'Pong') {
-          localStorage.setItem('auth:token', this.token)
+        if (result.data != null) {
+          localStorage.setItem('auth:token', result.data)
 
+          // Navigate to home
           window.location = '/'
         } else {
-          this.error = result.data
+          if (result.data == null) this.error = 'Invalid password'
```

Comment:
```suggestion
          this.error = 'Invalid password'
```

Isn't this branch always executed when `result.data` is `null`, because of the parent if statement?

Please use braces with your if statements. Prettify should have caught this.

... we have prettify right? that might be my next pr

Reply:
I left this code in as I found the previous code of displaying errors from the server to be helpful for users. You're right to mention that in this case the code is redundant; we can work on updating the routes to return appropriate HTTP codes so that errors and success messages are handled and displayed appropriately.

Reply:
Right... but logically this code does nothing vs just `this.error = 'Invalid password'`, why overcomplicate?